.github/workflows/repomix.yml

```
@@ -0,0 +1,80 @@
+name: Repomix Codebase Packaging
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)